cmd/compare_test.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"bytes"
+	"github.com/codilime/floodgate/test"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestCompare(t *testing.T) {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/auth/loggedOut", test.MockGateServerAuthLoggedOutHandler)
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("{}"))
+	})
+
+	ts := httptest.NewServer(mux)
+
+	dir, config, err := CreateTempFiles(ts.URL)
+	if err != nil {
+		t.Errorf("cmd.Compare() Error while creating temp config %v", err)
+	}
+	defer os.RemoveAll(dir)
```

Comment:
As discussed offline, add a safeguard so that if somehow dir is manipulated it won't result in data loss